edx/cookiecutter-django-ida
# Add edx-drf-extensions and corsheaders packages; include settings and…

… middleware for CORS, CSRF, and JWT Auth.

How to test:
* clone this repo, if necessary
* check out this branch
* `pip install cookiecutter; pip install edx-lint`
* `cd <workspace>`
* `cookiecutter path-to/cookiecutter-django-ida`
* fill things out at the prompts (don't use a hyphen in the `repo_name` value)
* create a test virtualenv and activate it
* run `make dev_requirements` from your `repo_name`
* run `./manage.py shell` to look at settings, or bring up the dev server, etc.
